os/booting-on-vmware.md

```diff
@@ -58,11 +85,99 @@ retained, although CoreOS supports config-drive on all platforms.
 For more information on customization that can be done with cloud-config, head
 on over to the [cloud-config guide][cloud-config guide].
 
-Note: The `$private_ipv4` and `$public_ipv4` substitution variables referenced
-in other documents are *not* supported on VMware.
+The `$private_ipv4` and `$public_ipv4` substitution variables are fully supported in cloud-config on VMware but only in CoreOS release 801.0.0 and greater. These releases has updated [coreos-cloudinit][coreos-cloudinit] v1.6.0 which allows you to configure network using [VMware GuestInfo options][vmware backdoor] and define `public` and `private` roles for ethernet interfaces.
+
+## VMware Backdoor
```

Comment:
VMWare Guestinfo Interface

(We are trying to eliminate use of the phrase "backdoor". See https://github.com/coreos/coreos-cloudinit/pull/404